packages/engine/Specs/Scene/MetadataComponentTypeSpec.js

```diff
@@ -402,9 +390,9 @@ describe("Scene/MetadataComponentType", function () {
     // Second result is max / 5
     // Third result is (max + 1) / 2
     const expectedResults = [
-      BigInt(0), // eslint-disable-line
-      BigInt(3689348814741910323), // eslint-disable-line
-      BigInt(9223372036854775808), // eslint-disable-line
+      BigInt(0),
+      BigInt("3689348814741910323"),
+      BigInt("9223372036854775808"),
```

Comment:
These were converted from number to strings. Was that intentional? I believe this is the test that's failing in CI too.